packages/core/src/serialization/v1/serialization.types.ts

```diff
@@ -58,13 +58,13 @@ export interface ISerializedBlockV1 {
  */
 export interface ISerializedConnectionV1 {
     /** The name of the block that the connection is to */
```

Comment:
Maybe update the comments to use "uniqueId" rather than "name". Or leave it if you think it's clear enough :)

Reply:
Ooh, good catch!  Missed it when doing the manual fix up of splitting the branches!